reporter/src/main/kotlin/model/DependencyTreeNode.kt

```diff
@@ -32,8 +32,6 @@ import org.ossreviewtoolkit.reporter.utils.ZeroBasedIntSequenceGenerator
  */
 @JsonIdentityInfo(property = "key", generator = ZeroBasedIntSequenceGenerator::class, scope = DependencyTreeNode::class)
 data class DependencyTreeNode(
-    val title: String,
-    @JsonInclude(JsonInclude.Include.NON_NULL)
     val linkage: PackageLinkage?,
```

Comment:
The `@JsonInclude(JsonInclude.Include.NON_NULL)` should not be removed.

Reply:
Lol, done.